department-of-veterans-affairs/caseflow
# Success banner fix in Scenario-3 Appeals-28703

Resolves (https://jira.devops.va.gov/browse/APPEALS-28703)

# Description
Fixed the Success banner message for Scenario-3 as shown in the FIGMA screens.

## Acceptance Criteria

## Testing Plan

1. Login as a Judge user to whom the case was assigned.
2. In the Your Cases page, under Assigned tab, select and click on the case and go to Case details page. 
<img width="836" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110078646/c7ac7fee-fd2e-4bc1-8c14-978a2546ab1c">

3. From the Currently active task, Actions dropdown, select Reassign to a judge
<img width="850" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110078646/36c82934-e74b-433e-971e-f022994ab741">
4. In the Assign task modal, select the Judge and give the details and click on Assign button
<img width="855" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110078646/945d7529-919e-4aaa-b110-a9bf4fdcceea">
5. Now you can see the Success banner 
<img width="858" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110078646/86a47be9-9bbd-4a72-a347-2a9e90aae82e">

 

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
